aikau/src/main/resources/alfresco/renderers/_PublishPayloadMixin.js

```diff
@@ -111,6 +111,42 @@ define(["dojo/_base/declare",
    return declare([ObjectProcessingMixin], {
 
       /**
+       * Whether to publish the topic globally
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       */
+      publishGlobal: false,
+
+      /**
+       * The payload to publish when this menu item is selected
+       *
+       * @instance
+       * @type {object}
+       * @default
+       */
+      publishPayload: null,
+
+      /**
+       * Whether to publish the topic using the parent pubSubScope
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       */
+      publishToParent: false,
+
+      /**
+       * The topic to publish when this menu item is selected
+       *
+       * @instance
+       * @type {string}
+       * @default
+       */
+      publishTopic: null,
+
```

Comment:
...but we probably want the processors and other attributes listed as well (see the various things that the PublishPayloadMixin can do)